extensions/npm/main.ts

```diff
@@ -56,7 +56,16 @@ if (Deno.args.length >= 1
     console.log(`[${green("phylum")}] Installing without build scripts…`);
 
     // Install packages without executing build scripts.
-    let status = await Deno.run({ cmd: ['npm', ...Deno.args, '--ignore-scripts']}).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'npm',
+        args: [...Deno.args, '--ignore-scripts'],
+        exceptions: {
+            write: ['~/.npm', './'],
+            run: ['npm'],
+            read: ['~/.npm', './'],
```

Comment:
I think these read permissions won't work with filesystem dependencies, like `npm i /tmp/test`. Pretty sure you need `read: true` for that.

Reply:
That's true, but having `read: true` and `net: true` would expose us to exfiltration (not through preinstall scripts at least, in this case). I still think it's ok.

Reply:
Usually we should allow read+net only without executing build scripts, then when we execute the build scripts we don't need `net` anymore. So that way we should be safe?

Reply:
Yeah, exactly.